shuffler/logic_to_aux.py

```diff
@@ -0,0 +1,85 @@
+import json
+from os import path
+from pathlib import Path
+import sys
+from typing import Any
+
+from _parser import clear_nodes, parse_area
+import click
+
+
+def logic_to_aux(logic_directory: str, output: str | None):
+    logic_files = list(Path(logic_directory).rglob("*.logic"))
+
+    for file in logic_files:
+        file_directory = file.relative_to(logic_directory)
+
+        rooms: dict[str, dict[str, Any]] = {}
+        with open(file, "r") as fd:
+            lines = [line.strip() for line in fd.readlines() if line and line.strip()]
+        clear_nodes()
+        nodes = parse_area(lines)
+
+        path_to_schema = "../"
+        directories_in_between = max(
+            str(file_directory).count("/"), str(file_directory).count("\\")
+        )
```

Comment:
```suggestion
        directories_in_between = str(file_directory.as_posix()).count("/")
```
`Path.as_posix` will ensure that a file path uses only forward slashes and not backslashes. Personally I find this easier to understand than using `max`, but I think either is fine